npc/cities/umbala.txt

```diff
@@ -1030,7 +1030,7 @@ OnTouch:
 		end;
 	case 3:
 		if (rand(1,2) == 2) {
-			percentheal -99,0;
+			Hp -= Hp-1;
```

Comment:
How about using `Hp = 1` since you reduce the Hp to 1 only ?

It would avoid the "calculation" part for `Hp - 1` and more simple to understand for those who are not familiar with coding.